webpack/module-minifier-plugin/src/ModuleMinifierPlugin.types.ts

```diff
@@ -23,6 +23,10 @@ export interface IModuleMinificationRequest {
    * File name to show for the source code in the source map
    */
   nameForMap: string | undefined;
+  /**
+   * Reserved variable names, e.g. __WEBPACK_EXTERNAL_MODULE_1__
```

Comment:
External variable names don't necessarily have numerical identifiers. That's handled, right?

Reply:
Yeah, the computation of the id is exactly the code used to generate them in webpack.